CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+Hence, this proposal promotes hard fork initiations as a standalone governance action, separate from protocol parameter updates, as in the Shelley scheme.
+
+### Treasury Withdrawals vs Project Catalyst
+
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one treasury withdrawal per epoch.
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+### The purpose of the DReps
+
+Nothing in this proposal limits the SPOs from becoming DReps.
+Why do we have DReps at all?
+SPOs are chosen purely for block production.
+Voters can choose to delegate their vote to DReps without needing to consider whether they are
+also a good block producer.
+
+### AVST (action voting stake threshold)
+
+The AVST exists to ensure that the votes of the DReps are meaningful.
+For example, if only 10 Lovelace were properly delegated to DReps, and 9 Lovelace "vote yes" to
+a governance action, we would doubt the legitimacy of the result since there are
+billions of ADA in circulation.
+A high enough AVST legitimizes the representation.
+
+### Ratification Requirements Table
+
+The requirements in the [ratification requirement table](#requirements) are explained here.
+Most of the governance actions have the same kind of requirements:
+the constitutional committee must reach quorum and the DReps must reach a sufficient number of
+'yes' votes, except in the situation where the AVST is too low, in which case the SPOs vote in
+place of the DReps.
+This includes these actions:
+* New Committee/quorum (normal state)
+* Update to the Constitution
+* Protocol parameter changes
+* Treasury withdrawal
+
+#### Motion of no-confidence
+
+A motion of no-confidence represents the lack of confidence by the Cardano community in the
+current Constitutional Committee, and hence the Constitutional Committee should have no
+say whatsoever in this governance action.
+In this situation, the SPOs and the DReps are left to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
+
+// TODO We need to fully understand the trade-offs between using the DReps votes with and without
+the AVST.
```

Comment:
A motion of no-confidence can potentially be malicious if it is initiated for reasons other than a genuine belief that the current Constitutional Committee is not acting in the best interests of the community or is not fulfilling its responsibilities effectively.

For example, if a group of stakeholders initiates a motion of no-confidence simply to disrupt the governance process or to pursue their own narrow interests, then this could be considered a malicious use of the mechanism.

Similarly, a motion of no-confidence initiated based on false information or rumors can also be considered malicious, as it undermines the integrity and effectiveness of the governance process.

It's important to have appropriate safeguards to prevent malicious or frivolous motions. This can help ensure that the mechanism is used in a responsible and legitimate way, and that the integrity and effectiveness of the governance process are maintained.

Reply:
> It's important to have appropriate safeguards to prevent malicious or frivolous motions. This can help ensure that the mechanism is used in a responsible and legitimate way, and that the integrity and effectiveness of the governance process are maintained.

What kind of safeguards do you have in mind? Are you thinking of something in the protocol, or something social/off-chain? It's important that we do not introduce a single point of failure.

Reply:
At the very least social/off-chain 🙂 more than anything, I thought this would be feedback that hasn’t been shared before. I’m happy if it leads to anything interesting.

Reply:
Gotcha. That's an extremely important point, thank you for bringing awareness to it.

Reply:
(Acknowledgment: I sent most of this CIP to ChatGPT using OpenAI’s newish chat API and it was ChatGPT who said that to me, so all credit goes to our machine overlords)

Reply:
Cheers 🍻

Reply:
> What kind of safeguards do you have in mind? Are you thinking of something in the protocol, or something social/off-chain? It's important that we do not introduce a single point of failure.

I would suggest an option to call for a pool operator vote to delay a motion of no-confidence. This delay should of course have a hard limit of 2-3 epochs. I think it is important that every party involved has enough time to prove their point.

Reply:
That sounds great!

Reply:
> I would suggest an option to call for a pool operator vote to delay a motion of no-confidence.

This is basically already built into the existing protocol that is proposed in this CIP, since the SPOs are a part of the vote for this action. In other words, they can simply chose to wait to vote.

Reply:
> This is basically already built into the existing protocol that is proposed in this CIP, since the SPOs are a part of the vote for this action. In other words, they can simply chose to wait to vote.

I see. What about resetting the tally once all the facts are on the table? Would the SPOs just let the action expire and create a new one?

Reply:
> What about resetting the tally once all the facts are on the table? Would the SPOs just let the action expire and create a new one?

yes, that would definitely work. The SPOs could abstain (either actively with an "abstain" vote, or by not voting) until they feel that they can make a decision, and if time runs out, always have the action submitted again.

Reply:
> and it was ChatGPT who said that to me

I'm just going to assume that the topic has already been discussed in a lot of places already on the internet and that ChatGPT is just regurgitating things. :)

Reply:
1 out of 10 times it shines a light from an angle I couldn’t see and it is totally worth it for it. Here’s a screenshot of the relevant part of the transcript. I had been asking it for comments and concerns. It pointed to malicious motions of no-confidence almost as a side thing, I had to ask for more info. Those moments feel spooky/enlightened 🙂
![40B35891-7F1E-4F91-BF3D-EA4B4EDED3EE](https://user-images.githubusercontent.com/417016/224578507-9f66c6eb-581b-49d4-977d-b6389d9a910a.jpeg)